### ISSUE TITLE
[a11y Design Review] Medical Device Reordering Tool design audit - Figma

### ISSUE THREAD

**Issue by rileyorr** [department-of-veterans-affairs/va.gov-team#96143]
> [!NOTE]
> **QUICK SUMMARY:** The Medical Device Reordering tool team just went through midpoint review and incorporated feedback into their designs prior to development. They'd like another accessibility design review to uncover any additional accessibility issues.

## Issue Description
This is an accessibility design audit of a Figma file

## About the Team

- **Team name**: Medical Device Reordering Tool
- **Product name:** Medical Device Reordering Tool
- **Product manager:**  N/A
- **Slack Channel:** N/A

### Supporting Artifacts 
- [Figma designs](https://www.figma.com/design/o9zkSuKTzHm9eQqHTEN1iz/Medical-supplies-reorder?node-id=1164-12192&t=BsCYo8NQNvNX4Qhm-1)

```[tasklist]
### a11y Tasks
- [x] Review the designs using the CAIA Figma audit checklist
- [x] Provide comments on Figma file as needed
- [x] Provide final feedback and audit file in this ticket as a comment
```
```[tasklist]
### Acceptance Criteria
- [x] Designs have been reviewed and feedback has been shared with the team
```
```[tasklist]
## Ticket Updates (CAIA Internal)
- [ ] _Connect to an `Epic` or `Intake` (what body of work is this a part of?)_
- [ ] _Label with `Originator/Team` (product team or stakeholder requesting support)_
- [ ] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Naomi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [ ] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
```

**Comment by rileyorr** [comment]
I created [an audit document](https://docs.google.com/document/d/1Au0IVbXwFBhXLO6zJPLbKec754VTByTkTJ3tHPRFIlg/edit?usp=sharing) to track the review tasks that were completed and any issues or recommendations that the team should consider remediating. 

The findings and recommendations are also below: 

### Issue: Consider increasing spacing between touch targets
**Screen:** [Contact information](https://www.figma.com/design/o9zkSuKTzHm9eQqHTEN1iz/Medical-supplies-reorder?node-id=2139-27305&t=whSHrn7stVwgXbct-1)
**Version:** All
**Experience standard:** N/A
**Description:** The ‘Edit’ link in the shipping address component has 18px spacing between it and the ‘Finish this order later’ link. Although links don’t need to pass [SC 2.5.8 Touch Size](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html), you might consider increasing the spacing between the two links to 24px so users, including those with mobility issues such as tremors, have a lesser chance of accidentally selecting the incorrect element. Element spacing on other art boards were checked and have 24px spacing, so this would also provide consistency across pages.
![A screenshot from the Figma file showing a mocked up shipping address, edit link, and link to finish the order later.](https://github.com/user-attachments/assets/d59f558c-a648-4770-a31e-9b9ce4979ddb)
**Recommendation:** Consider increasing the spacing between the ‘Edit’ and ‘Finish this order later’ elements to 24px.

### Issue: Consider asking a question or prompting Veterans to select supplies available to reorder
**Screen:** [Select supplies](https://www.figma.com/design/o9zkSuKTzHm9eQqHTEN1iz/Medical-supplies-reorder?node-id=2698-24575&t=whSHrn7stVwgXbct-1)
**Version:** All
**Experience standard:** [User has enough information to complete a task](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**Description:** Although all the supplies available for reorder are grouped under the ‘Available for reorder’ header, you might consider adding a label prompting the Veteran to select items for reorder or asking what items they want to reorder to make it clearer what action needs to be taken. Brian shared similar feedback in [his Midpoint review](https://github.com/department-of-veterans-affairs/va.gov-team/issues/90746) response.
![A screenshot from the Figma file of a mocked up list of supplies available to order.](https://github.com/user-attachments/assets/02133ab3-6477-425c-9a09-aa760f0e1993)
**Recommendation:** Consider adding a label prompting the Veteran to select items for reorder or asking what items they want to reorder to make it clearer what action needs to be taken.

### Issue: Consistent supply availability status messages
**Screen:** [Intro page with supplies](https://www.figma.com/design/o9zkSuKTzHm9eQqHTEN1iz/Medical-supplies-reorder?node-id=2698-24746&t=whSHrn7stVwgXbct-1)
**Version:** All
**Experience standard:** [User encounters accurate, consistent content](https://design.va.gov/about/experience-standards/#user-encounters-accurate-consistent-content)
**Description:** There is inconsistent content where two unavailable supply cards have the same status. There’s [a Figma comment](https://www.figma.com/design/o9zkSuKTzHm9eQqHTEN1iz?node-id=1164-12192#1005641690) related to this where I recommended looping Laura Willwerth to provide feedback on the status’ content.
![A screenshot from the Figma file of a list of supplies that are unavailable to order.](https://github.com/user-attachments/assets/f46058f1-85f7-48df-a27a-5b7219cf5ac7)
**Recommendation:** Work with Laura Willwerth on drafting plain language content for the status related to supplies that haven’t been reordered in the past 2 years. I also recommend having her review all the design’s art boards from a content perspective. 

@Jenny-yw overall this is in a great place heading into development! I just had some items to consider. Let me know if you have any questions! I'd be happy to chat about any of the feedback. Looking forward to the staging audit and upcoming AT usability study!

**Comment by rileyorr** [comment]
@sara-amanda and @NaomiPMC I moved this ticket to validate since the work is complete!
